### ISSUE TITLE
Custom directory for new SF service

### ISSUE THREAD

**Issue by Zonciu** [Azure/service-fabric-issues#757]
How can I set directory while creating new service? 
Project structure example:
```
MyProject
├─src
│   ├─Service1
│   └─Service2
├─test
│   ├─Service1Test
│   └─Service2Test
└─ServiceFabric
     └─MyApp
```
Everytime I create a new service, I have to `create a new service(generated in ServiceFabric dir)`-`remove it`-`move to src dir`-`add to solution`-`add to SF application service`.

**Comment by dbreshears** [comment]
@Zonciu ,  thanks for the feedback.    We are working on adding support for selecting the folder in the add new service dialog.   Hope to have this in an upcoming tooling release.

**Comment by dbreshears** [comment]
@Zonciu,   support for this was released recently in VS2017 15.6 release

**Comment by Zonciu** [comment]
@dbreshears Thanks! I saw it.
